pkg/target/kinesis/kinesis.go

```diff
@@ -44,7 +43,7 @@ func (c *client) Send(result v1alpha2.PolicyReportResult) {
 
 	body := new(bytes.Buffer)
 
-	if err := json.NewEncoder(body).Encode(http.NewJSONResult(result)); err != nil {
+	if err := json.NewEncoder(body).Encode(result); err != nil {
```

Comment:
Hey, thanks for this fix. This change changes the structure of the json compared to previous versions. I would prefer to leave this as it is to be backward compatible

Reply:
ok then, I'll work on changes to `type Result struct` and get back to you tomorrow.

Reply:
@fjogeleit done, can we merge it or something else needed?

Reply:
looks good to me, can merge it